test/functional/api/vim_spec.lua

```diff
@@ -1770,6 +1770,11 @@ describe('API', function()
     end)
 
     it('validation', function()
+      eq("Unknown option 'foobar'", pcall_err(api.nvim_get_option_value, 'foobar', {}))
+      eq(
+        "Unknown option 'foobar'",
+        pcall_err(api.nvim_get_option_value, 'foobar', { win = api.nvim_get_current_win() })
+      )
```

Comment:
The assertion is only triggered for `nvim_set_option_value`, not for `nvim_get_option_value`.